### PR TITLE
Update vertexai changelog file

### PR DESCRIPTION
**Pull request by rlazo**
It includes the full list of changes since the last release.

### CODE REVIEW

File: firebase-vertexai/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
 # Unreleased
-* [changed] Breaking Change: refactored enum classes to be normal classes (#6340). 
+* [changed] Breaking Change: introduced `Citations` class. Now `CitationMetadata` wraps that type.
```
Comment (daymxn) [inline firebase-vertexai/CHANGELOG.md:2]: Should this have a PR link?

**Review by daymxn — COMMENTED** [review]
+1, but I defer to @rachelsaunders for final approval

File: firebase-vertexai/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+* [changed] Breaking Change: removed `UNSPECIFIED` option of the `HarmBlockThreshold`, `HarmProbability`, `HarmSeverity`, and `BlockReason` enums (#6294).
+* [feature] Added support for `title`, and `publicationDate` in Citations (#6309).
+* [feature] Added support for `frequencyPenalty`, `presencePenalty`, and `HarmBlockMethod` (#6309).
+* [changed] Breaking Change: renamed all types and methods starting with `blob` to start with `inlineData` (#6309).
+* [changed] Breaking Change: marked `GenerativeModel` properties private (#6309).
+* [changed] Breaking Change: introduced `FunctionCall` and `FunctionResponse` types. Now `FunctionCallPart` and `FunctionResponsePart` wrap those types respectively (#6311).
+* [changed] Breaking Change: renamed `BlockThreshold` as `HarmBlockThreshold` (#6262).
+* [changed] Breaking Change: replaced sealed classes with abstract classes for `StringFormat`(#6334).
+* [changed] Breaking Change: changed order of arguments in `InlineDataPart` to match `ImagePart`(#6340).
+* [changed] Breaking Change: refactored enum classes to be normal classes (#6340).
```
Comment (rachelsaunders) [inline firebase-vertexai/CHANGELOG.md:18]:
```suggestion
* [feature] Added support for `title` and `publicationDate` in citations. (#6309)
* [feature] Added support for `frequencyPenalty`, `presencePenalty`, and `HarmBlockMethod`. (#6309)
* [changed] **Breaking Change**: Introduced `Citations` class. Now `CitationMetadata` wraps that type.
* [changed] **Breaking Change**: Introduced `FunctionCall` and `FunctionResponse` types. Now `FunctionCallPart` and `FunctionResponsePart` wrap those types, respectively. (#6311)
* [changed] **Breaking Change**: Reworked `Schema` declaration mechanism. (#6258)
* [changed] **Breaking Change**: Reworked function calling mechanism to use the new `Schema` format. Function calls no longer use native types, nor include references to the actual executable code. (#6258)
* [changed] **Breaking Change**: Made `totalBillableCharacters` field in `CountTokens` nullable and optional. (#6294)
* [changed] **Breaking Change**: Removed `UNKNOWN` option for the `HarmBlockThreshold` enum. (#6294)
* [changed] **Breaking Change**: Removed `UNSPECIFIED` option for the `HarmBlockThreshold`, `HarmProbability`, `HarmSeverity`, and `BlockReason` enums. (#6294)
* [changed] **Breaking Change**: Renamed `BlockThreshold` as `HarmBlockThreshold`. (#6262)
* [changed] **Breaking Change**: Renamed all types and methods starting with `blob` to start with `inlineData`. (#6309)
* [changed] **Breaking Change**: Changed the order of arguments in `InlineDataPart` to match `ImagePart`.(#6340)
* [changed] **Breaking Change**: Changed `RequestOption` to accept only `long` timeout values. (#6289)
* [changed] **Breaking Change**: Moved `requestOptions` to the last positional argument in the `generativeModel` argument list. (#6292)
* [changed] **Breaking Change**: Replaced sealed classes with abstract classes for `StringFormat`. (#6334)
* [changed] **Breaking Change**: Refactored enum classes to be normal classes. (#6340)
* [changed] **Breaking Change**: Marked `GenerativeModel` properties as private. (#6309)
```

**Review by rachelsaunders — CHANGES_REQUESTED** [review]

File: firebase-vertexai/CHANGELOG.md (hunk 1 above)
Comment (rlazo) [reply firebase-vertexai/CHANGELOG.md]: done

**Review by daymxn — APPROVED** [review]

File: firebase-vertexai/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,21 @@
...
+* [changed] **Breaking Change**: Introduced `FunctionCall` and `FunctionResponse` types. Now `FunctionCallPart` and `FunctionResponsePart` wrap those types, respectively. (#6311)
+* [changed] **Breaking Change**: Reworked `Schema` declaration mechanism. (#6258)
+* [changed] **Breaking Change**: Reworked function calling mechanism to use the new `Schema` format. Function calls no longer use native types, nor include references to the actual executable code. (#6258)
+* [changed] **Breaking Change**: Made `totalBillableCharacters` field in `CountTokens` nullable and optional. (#6294)
+* [changed] **Breaking Change**: Removed `UNKNOWN` option for the `HarmBlockThreshold` enum. (#6294)
+* [changed] **Breaking Change**: Removed `UNSPECIFIED` option for the `HarmBlockThreshold`, `HarmProbability`, `HarmSeverity`, and `BlockReason` enums. (#6294)
+* [changed] **Breaking Change**: Renamed `BlockThreshold` as `HarmBlockThreshold`. (#6262)
+* [changed] **Breaking Change**: Renamed all types and methods starting with `blob` to start with `inlineData`. (#6309)
+* [changed] **Breaking Change**: Changed the order of arguments in `InlineDataPart` to match `ImagePart`.(#6340)
```
Comment (rachelsaunders) [inline firebase-vertexai/CHANGELOG.md:13]:
```suggestion
* [changed] **Breaking Change**: Changed the order of arguments in `InlineDataPart` to match `ImagePart`. (#6340)
```

**Review by rachelsaunders — APPROVED** [review]

**Pull request merged**